Azure.ResourceManager.Core/ResourceGroupOperations.cs

```diff
@@ -27,11 +28,20 @@ public class ResourceGroupOperations : ResourceOperationsBase<ResourceGroup>,
         /// </summary>
         /// <param name="options"> The client parameters to use in these operations. </param>
         /// <param name="rgName"> The name of the resource group to use. </param>
+        /// <exception cref="ArgumentException"> rgName cannot be longer than 90 characters. </exception>
```

Comment:
move these to the public methods that call this otherwise customers won't see the hints